pkg/apis/sources/test_helper.go

```diff
@@ -14,11 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha2
+package sources
```

Comment:
Not sure whether it's ok that we move the test_helper.go directly under pkg/apis/sources

Reply:
the thing is that will be shipped and not treated as part of UTs.
I think we might just need to duplicate code. 
Even in v1alpha2, test_helper.go is not treated as UT, isn't it? I think is part of the binary?

Reply:
How about we put it in v1 and other packages just use the test_helper.go in v1? Just wanna following the comment in this PR https://github.com/knative/eventing/pull/3407#discussion_r445205903

Reply:
Yep. In v1alpha2, test_helper.go is not treated as UT and it's part of the binary.